### PR TITLE
Fix to INSTALL.md and installation scripts

### PR DESCRIPTION
**Pull request by didriklundberg**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ This guide assumes a fresh install of Ubuntu 22.04.
 
 ## Dependencies
 
-* HOL4 
-* Ott
+* HOL4
 * Petr4
 * Git
 * Python 3
```
Comment (palmskog) [inline INSTALL.md:24]: Isn't this optional also now that Ott is optional? Or where are you using Python except for post-processing Ott output?

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -38,16 +38,7 @@ jobs:
...
-          ./.github/scripts/install_hol4.sh
-
-      - name: Prepare OCaml
-        uses: ocaml/setup-ocaml@v2
-        with:
-          ocaml-compiler: 4.13.x
-
-      - name: Prepare Ott
-        run: opam install ott
+          ./scripts/install.sh && ./scripts/install2.sh
```
Comment (palmskog) [inline .github/workflows/build.yaml:41]: Maybe good to use something more descriptive than "install" and "install2".

**Review by palmskog — APPROVED** [review]
Fine by me, feel free to commit directly into the `ott-inductive` branch instead.

File: INSTALL.md (hunk 1 above)
Comment (didriklundberg) [reply INSTALL.md:24]: Fixed.

**Pull request merged**